src/framework/nunit.framework-4.5.csproj

```diff
@@ -369,6 +369,10 @@
     <Compile Include="Constraints\ExceptionTypeConstraint.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="FSharp.Core, Version=4.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
We shouldn't be pulling language specific assemblies into the framework. There is no guarantee consumers of NUnit even have F# installed. I assume this doesn't ship with the core framework based on the hintpath?